### ISSUE TITLE
ML: Classification

### ISSUE THREAD

**Issue by philusnarh** [philusnarh/PROJECT#22]
https://www.kaggle.com/elmadj/detect-credit-card-fraud-using-logistic-regression/notebook
https://www.kaggle.com/theonarh/kernels/notebooks/new?forkParentScriptVersionId=2167747
https://www.data-blogger.com/2017/06/15/fraud-detection-a-simple-machine-learning-approach/

**Comment by philusnarh** [comment]
**7 Types of Regression Techniques you should know!**
https://www.analyticsvidhya.com/blog/2015/08/comprehensive-guide-regression/

**A Complete Tutorial to Learn Data Science with Python from Scratch**
https://www.analyticsvidhya.com/blog/2016/01/complete-tutorial-learn-data-science-python-scratch-2/
https://www.analyticsvidhya.com/blog/2015/10/regression-python-beginners/
https://www.analyticsvidhya.com/blog/2016/01/complete-tutorial-ridge-lasso-regression-python/

**Comment by philusnarh** [comment]
**weather**
https://stackabuse.com/using-machine-learning-to-predict-the-weather-part-1/

**Comment by philusnarh** [comment]
https://drive.google.com/open?id=17Ervh9-3A8S4EpvKsGq3pnOpL1r6lqQr

**Comment by philusnarh** [comment]
**X-ray Predictions**
https://www.kaggle.com/paultimothymooney/predicting-pathologies-in-x-ray-images
https://github.com/brucechou1983/CheXNet-Keras

**image segementation**
https://github.com/Borda/pyImSegm

**Comment by philusnarh** [comment]
**tensor flow installation**
http://deeplearning.lipingyang.org/2017/01/19/install-tensorflow-for-python-2-7-and-3-5-on-one-machine/

**Comment by philusnarh** [comment]
**Softmax vs Sigmoid function in Logistic classifier?**
https://stats.stackexchange.com/questions/233658/softmax-vs-sigmoid-function-in-logistic-classifier

**Comment by philusnarh** [comment]
**Natural Language processing**

https://github.com/tensorflow/workshops/blob/master/extras/keras-bag-of-words/keras-bow-model.ipynb

**Comment by philusnarh** [comment]
**Road sign classification**

https://chsasank.github.io/keras-tutorial.html

**Comment by philusnarh** [comment]
**Transfer Learning**
https://medium.com/@14prakash/transfer-learning-using-keras-d804b2e04ef8

**Comment by philusnarh** [comment]
**Keras Metrics**
https://machinelearningmastery.com/custom-metrics-deep-learning-keras-python/

**Comment by philusnarh** [comment]
**Plotting Feature Importances**
https://www.kaggle.com/grfiv4/plotting-feature-importances

**Comment by philusnarh** [comment]
**Your First Machine Learning Model**
https://www.kaggle.com/dansbecker/your-first-machine-learning-model
https://www.kaggle.com/kanncaa1/machine-learning-tutorial-for-beginners

**cats and dog using cnn**
https://pythonprogramming.net/convolutional-neural-network-kats-vs-dogs-machine-learning-tutorial/
https://becominghuman.ai/building-an-image-classifier-using-deep-learning-in-python-totally-from-a-beginners-perspective-be8dbaf22dd8
Understanding CNN
http://cs231n.github.io/convolutional-networks/

**flight delays**
https://perso.telecom-paristech.fr/qleroy/aml/lab3.html
https://www.datasciencecentral.com/profiles/blogs/predicting-flights-delay-using-supervised-learning
https://www.kaggle.com/fabiendaniel/predicting-flight-delays-tutorial
https://github.com/Sudeepv5/flight-delay-prediction
https://www.kaggle.com/levaniz/machine-learning-analysis-of-flights-data

**Comment by philusnarh** [comment]
**Weather Forecasting**
https://towardsdatascience.com/random-forest-in-python-24d0893d51c0
https://datascience.stackexchange.com/questions/32888/using-machine-learning-to-predict-temperature

**Comment by philusnarh** [comment]
**AstroImage Mapping**
https://github.com/rflamary/AstroImageReconsCNN

**Comment by philusnarh** [comment]
**Research potential**
https://www.kaggle.com/farazrahman/predicting-star-galaxy-quasar-with-svm/notebook?utm_medium=email&utm_source=mailchimp&utm_campaign=datanotes-20181004

**Face Recognition on Olivetti Dataset**
https://www.kaggle.com/serkanpeldek/face-recognition-on-olivetti-dataset

**Datascientist_selary_analysis**
https://www.kaggle.com/scarecrow88/datascientist-selary-analysis

**Anomaly Detection - Credit Card Fraud Analysis**
https://www.kaggle.com/pavansanagapati/anomaly-detection-credit-card-fraud-analysis

**YouTube Trending Videos Analysis**
https://www.kaggle.com/ammar111/youtube-trending-videos-analysis

**Your First Machine Learning Model**
https://www.kaggle.com/dansbecker/your-first-machine-learning-model

**HousePrice_Predic**
https://www.kaggle.com/klauslyu/houseprice-predict

**Intro to Deep Learning and Computer Vision**
https://www.kaggle.com/dansbecker/intro-to-deep-learning-and-computer-vision

**Film recommendation engine**
https://www.kaggle.com/fabiendaniel/film-recommendation-engine

**Seaborn Tutorial for Beginners**
https://www.kaggle.com/kanncaa1/seaborn-tutorial-for-beginners

**Deep Learning Tutorial for Beginners**
https://www.kaggle.com/kanncaa1/deep-learning-tutorial-for-beginners

**Python Programming from A to Z**
https://www.kaggle.com/dark4user/python-programming-from-a-to-z

**A Tensorflow Keras CNN approach**
https://www.kaggle.com/amneves/a-tensorflow-keras-cnn-approach/notebook

**Honey Bee health detection with CNN**
https://www.kaggle.com/dmitrypukhov/honey-bee-health-detection-with-cnn?utm_medium=email&utm_source=mailchimp&utm_campaign=datanotes-20181004

**Predicting flight delays [Tutorial]**
https://www.kaggle.com/fabiendaniel/predicting-flight-delays-tutorial
https://www.kaggle.com/niranjan0272/us-flight-delay

**Telecom Customer Churn Prediction**
https://www.kaggle.com/pavanraj159/telecom-customer-churn-prediction/data

**Comment by philusnarh** [comment]
**kaggle kernels**
https://www.kaggle.com/kernels

**Comment by philusnarh** [comment]
**https://machinelearningmastery.com/ensemble-machine-learning-algorithms-python-scikit-learn/**

**Comment by philusnarh** [comment]
**Financial Distress Prediction: Bankruptcy Prediction**
https://www.kaggle.com/saurabhharsh/financial-distress-prediction/notebook
https://www.kaggle.com/eric2396/financial-distress-prediction-using-ann
https://www.kaggle.com/caspitush/playing-with-pca
https://www.kaggle.com/keigito/financial-distress-prediction-with-regression
https://www.kaggle.com/adityasheth/financial-distress-prediction
https://www.kaggle.com/azurtheowl/distress-prediction

**weather**
https://github.com/prl900/DeepWeather
https://jaxenter.com/convolutional-lstm-deeplearning4j-146157.html
CNN Architectures: VGG, Resnet, Inception, Alex

**CNN Architectures: VGG, Resnet, Inception, Alex**
https://www.kaggle.com/shivamb/cnn-architectures-vgg-resnet-inception-alex

**radar**
https://github.com/wqxu/ConvLSTM

**Comment by philusnarh** [comment]
**Waste Management Facilities**

https://www.kaggle.com/new-york-state/nys-solid-waste-management-facilities
https://www.kaggle.com/c/waste-classification
https://www.kaggle.com/data/40778
https://www.kaggle.com/jboysen/austin-waste/kernels
https://www.kaggle.com/tsansom/eda-of-austin-waste

**Comment by philusnarh** [comment]
**Crime Detection Using ML**
https://www.kaggle.com/fahd09/eda-of-crime-in-chicago-2005-2016
https://www.kaggle.com/ravitejayerra/chicago-crime-data-analysis
https://www.kaggle.com/currie32/crimes-in-chicago/kernels
https://www.kaggle.com/threadid/chicago-crimes-regression-neural-network/data

**Comment by philusnarh** [comment]
**Data Exploration**
https://www.kaggle.com/artgor/exploration-of-data-step-by-step

**Linear Regression from Scratch**
https://www.kaggle.com/kennethjohn/linear-regression-from-scratch

**A Comprehensive ML Workflow with Python**
https://www.kaggle.com/mjbahmani/a-comprehensive-ml-workflow-with-python

**Adding an xgboost model**
https://www.kaggle.com/apapiu/regularized-linear-models

**Machine Learning Tutorial for Beginners**
https://www.kaggle.com/kanncaa1/machine-learning-tutorial-for-beginners

**Deep Learning Tutorial for Beginners**
https://www.kaggle.com/kanncaa1/deep-learning-tutorial-for-beginners

**Comment by philusnarh** [comment]
**Estimating Rainfall From Weather Radar Readings Using Recurrent Neural Networks**
https://simaaron.github.io/Estimating-rainfall-from-weather-radar-readings-using-recurrent-neural-networks/

**Multivariate Time Series Forecasting with LSTMs in Keras**
https://machinelearningmastery.com/multivariate-time-series-forecasting-lstms-keras/

**Student Performance Analysis**
https://www.kaggle.com/jshen97/student-performance-analysis-dt-updated/data

**New Weather Forecasting**
-- Prediction of Rainfall
https://www.kaggle.com/nasirmeh/prediction-of-rainfall
https://www.kaggle.com/shwetabh123/rainfall-prediction

**Comment by philusnarh** [comment]
Introduction
I’ve always had a passion for learning and consider myself a lifelong learner. Being at SAS, as a data scientist, allows me to learn and try out new algorithms and functionalities that we regularly release to our customers. Often times, the algorithms are not technically new, but they’re new to me which makes it a lot of fun.

Recently, I had the opportunity to learn more about t-Distributed Stochastic Neighbor Embedding (t-SNE). In this post I’m going to give a high-level overview of the t-SNE algorithm. I’ll also share some example python code where I’ll use t-SNE on both the Digits and MNIST dataset.

What is t-SNE?
t-Distributed Stochastic Neighbor Embedding (t-SNE) is an unsupervised, non-linear technique primarily used for data exploration and visualizing high-dimensional data. In simpler terms, t-SNE gives you a feel or intuition of how the data is arranged in a high-dimensional space. It was developed by Laurens van der Maatens and Geoffrey Hinton in 2008.

t-SNE vs PCA
If you’re familiar with Principal Components Analysis (PCA), then like me, you’re probably wondering the difference between PCA and t-SNE. The first thing to note is that PCA was developed in 1933 while t-SNE was developed in 2008. A lot has changed in the world of data science since 1933 mainly in the realm of compute and size of data. Second, PCA is a linear dimension reduction technique that seeks to maximize variance and preserves large pairwise distances. In other words, things that are different end up far apart. This can lead to poor visualization especially when dealing with non-linear manifold structures. Think of a manifold structure as any geometric shape like: cylinder, ball, curve, etc.

t-SNE differs from PCA by preserving only small pairwise distances or local similarities whereas PCA is concerned with preserving large pairwise distances to maximize variance. Laurens illustrates the PCA and t-SNE approach pretty well using the Swiss Roll dataset in Figure 1 [1]. You can see that due to the non-linearity of this toy dataset (manifold) and preserving large distances that PCA would incorrectly preserve the structure of the data.

Read more ... https://www.kdnuggets.com/2018/08/introduction-t-sne-python.html

**Comment by philusnarh** [comment]
**Supervised Machine Learning: Classification**
https://towardsdatascience.com/supervised-machine-learning-classification-5e685fe18a6d


PyAstronomy
https://www.hs.uni-hamburg.de/DE/Ins/Per/Czesla/PyA/PyA/funcFitDoc/tutorialMCMC.html

**Comment by philusnarh** [comment]
**AutoEncoder**

https://nbviewer.jupyter.org/urls/dl.dropbox.com/s/2vous47vib4a55o/autoencoder1.ipynb#

https://nbviewer.jupyter.org/urls/dl.dropbox.com/s/oflj51a252tig83/image_denoising_c_autoencoder.ipynb
